File: Web/AFSport.Web.Core/Model/Pedido.cs
using System;
using System.Collections.Generic;
using AFSport.Web.Core.Enum;

namespace AFSport.Web.Core.Model
{
    public class Pedido
    {
        public Pedido()
        {

        }

        public int IdPedido { get; set; }
        public DateTime Data { get; set; }
        public int IdCliente { get; set; }
        public Cliente Cliente { get; set; }
        public int IdUsuario { get; set; }
        public Usuario Usuario { get; set; }
        public PedidoStatus Status { get; set; }
    }
}

File: Painel/src/app/pedidos/pedido/pedido.ts
import { Cliente } from "src/app/clientes/cliente/cliente";
import { Usuario } from "src/app/usuarios/usuario/usuario";
import { PedidoStatus } from "./status.enum";

export interface Pedido {
    idPedido: number,
    data: Date,
    idCliente: number,
    cliente: Cliente,
    idUsuario: number,
    usuario: Usuario,
    status: PedidoStatus,
}